### PR TITLE
[11.0][ADD]l10n_it_corrispettivi_fatturapa_out

### PR DESCRIPTION
**Pull request by GSLabIt**
Backport from 12.0

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (SimoRubi) [inline .vscode/settings.json]: remove file

File: l10n_it_corrispettivi_fatturapa_out/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
+==========================================================
```
Comment (SimoRubi) [inline l10n_it_corrispettivi_fatturapa_out/README.rst]: regenerate readme, here links are still for 12.0

File: .vscode/settings.json (hunk 1 above)
Comment (GSLabIt) [reply .vscode/settings.json]: wooopsss ..forgot .gitignore, will fix

File: l10n_it_corrispettivi_fatturapa_out/README.rst (hunk 2 above)
Comment (GSLabIt) [reply l10n_it_corrispettivi_fatturapa_out/README.rst]: how to regenerate? 😬

Comment (SimoRubi) [reply l10n_it_corrispettivi_fatturapa_out/README.rst]: The script is https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py, follow the instructions in https://github.com/OCA/maintainer-tools to install and https://github.com/OCA/maintainer-tools#readme-generator to use it.

**Comment by GSLabIt** [comment]
@SimoRubi credo/spero di aver fatto tutto correttamente 😬

File: l10n_it_corrispettivi_fatturapa_out/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+**Italiano**
+
+Questo modulo integra le funzionalità dei corrispettivi con la generazione della fattura elettronica.
+
+**English**
+
+This module integrates corrispettivi and electronic invoicing.
```
Comment (primes2h) [inline l10n_it_corrispettivi_fatturapa_out/readme/DESCRIPTION.rst]: Vedi https://github.com/OCA/l10n-italy/pull/977#discussion_r255153839

Magari attendi quelle modifiche e poi riportale qui.

Comment (GSLabIt) [reply l10n_it_corrispettivi_fatturapa_out/readme/DESCRIPTION.rst]: Ok, sembra comunque che ci sia anche un bug; il pulsante viene mostrato comunque se visualizzi la fattura passando da SO smart btn.

**Comment by GSLabIt** [comment]
To much confusion ..will reopen

**Comment by tafaRU** [comment]
Superseeded by https://github.com/OCA/l10n-italy/pull/983

**Pull request closed without merging**